src/components/transactions/filters/dateFieldGroup.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines */
```

Comment:
😢 
Maybe you can save some lines by replacing `this.handleClickOutsideDropdown` with use of `DropdownButton` 😉 
Or open a refactoring ticket for this file.

Reply:
I think that the creating a refactor ticket is better, so this PR doesn't go to far from the original scope.

Reply:
issue created: #2360